src/Modals.tsx

```diff
@@ -28,13 +28,14 @@ export const Modals = () => {
     isNotificationPwaModalOpen
   } = useModals()
   const { close: closeWeb3Modal } = useWeb3Modal()
+  const { open: isWeb3ModalOpen } = useWeb3ModalState()
 
   const { notifyRegisterMessage, notifyRegisteredKey, userPubkey } = useContext(W3iContext)
 
   const notificationsEnabled = useNotificationPermissionState()
 
   const notificationModalClosed = checkIfNotificationModalClosed()
-  const explicitlyDeniedOnDesktop = !isMobile() && window.Notification?.permission === 'denied'
+  const explicitlyDeniedOnDesktop = !isMobile() && window.Notification.permission === 'denied'
```

Comment:
Why remove the `?`, access to `Notification` directly breaks the app in certain mobile environments

Reply:
This was an ESLint autofix I believe, will undo it

Reply:
Added it back and removed the ESLint error for that expression so peoples' editors don't auto fix in future.